### ISSUE TITLE
Generate video thumbnail from recorded video

### ISSUE THREAD

**Issue by preveenraj** [0x006F/react-media-recorder#70]
Hey,
Is it feasible to bring in a feature to generate video thumbnail from a recorded video and get it as a jpg/png file so that it can be stored along with the media metadata for better user experience?

**Comment by 0x006F** [comment]
generating thumbnails for a video snip is very much out of scope from this component's context. This would increase the complexity of this. Just want to keep things as simple as possible.

**Comment by preveenraj** [comment]
Yeah, understood. If it were you, how would you generate a thumbnail from a video snip. I saw many solutions in the web. Curious. @0x006F
